examples/whiteboard/public/main.js

```diff
@@ -20,7 +20,7 @@
   //Touch support for mobile devices
   canvas.addEventListener('touchstart', onMouseDown, false);
   canvas.addEventListener('touchend', onMouseUp, false);
-  canvas.addEventListener('touchcancle', onMouseUp, false);
+  canvas.addEventListener('touchcancel', onMouseUp, false);
```

Comment:
Hi, @shirtleton
Updated the event `touchcancle` to `touchcancel`.
Thanks for pointing out in last review.
Hope, the pull request can be merged now.